src/index.ts

```diff
@@ -274,7 +274,12 @@ const getNamedType = (opts: Options<NamedTypeNode>): string | number | boolean =
                         );
                         const enumConverter = createNameConverter(opts.enumValuesConvention, opts.transformUnderscore);
                         const value = foundType.values ? foundType.values[0] : '';
-                        return `${typenameConverter(foundType.name, opts.enumsPrefix)}.${enumConverter(value)}`;
+                        return handleValueGeneration(
+                            opts,
+                            undefined,
+                            () => `${typenameConverter(foundType.name, opts.enumsPrefix)}.${enumConverter(value)}`,
+                        );
+                        // return `${typenameConverter(foundType.name, opts.enumsPrefix)}.${enumConverter(value)}`;
```

Comment:
you can remove this comment

Reply:
Woops - left this in during TDD. Will update.

Reply:
This is done 😃